frontend/src/App.js

```diff
@@ -1,27 +1,31 @@
-import React from "react";
-import { Route, BrowserRouter as Router, Routes } from "react-router-dom";
-import logo from "./logo.svg";
-import "./App.css";
-import Footer from "./components/Footer.js";
-import LandingPage from "./components/LandingPage.js";
-import Navbar from "./components/Navbar.js";
-import About from "./pages/About.js";
-import Home from "./pages/Home.js";
+import './App.css';
+import React from 'react';
+import { Route, BrowserRouter as Router, Routes } from 'react-router-dom';
+import Footer from './components/Footer.js';
+import LandingPage from './components/LandingPage.js';
+import Navbar from './components/Navbar.js';
+import About from './pages/About.js';
+import Home from './pages/Home.js';
+import SignUp from './components/SignUp/SignUp.jsx';
+import { BrowserRouter } from 'react-router-dom';
+import LoginForm from './components/LoginForm/LoginForm.jsx';
 import ChangeInfo from "./Change/ChangeInfo";
 
+
 const App = () => {
   return (
-    <Router>
-      <div className="App">
-        <Navbar />
-        <LandingPage />
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/changeInfo" element={<ChangeInfo />} />
-        </Routes>
-        <Footer />
-      </div>
-    </Router>
+    <BrowserRouter>
+      <Routes>
+        <Route index element={<LandingPage />} />
+        <Route path="/about" element={<About />} />
+        <Route path="/" element={<LandingPage />} />
+        <Route path="/sign-up" element={<SignUp />} />
+        <Route path="/home" element={<Home />} />
+        <Route path="log-in" element={<LoginForm />} />
+        <Route path="Change" element=(<ChangeInfo />} />
```

Comment:
Nice adding the change route! Is it possible for you to make it lowercase to be consistent with the other routes?